src/Rules/Library/ChromiumComponentsShouldUseWebScanner.cs

```diff
@@ -27,7 +25,7 @@ public override bool PassesTest(IA11yElement e)
 
         protected override Condition CreateCondition()
         {
-            return Chrome & (Document | AnyAncestor(Document));
+            return IsChromiumDocument;
```

Comment:
This is the change that limits `ChromiumComponentsShouldUseWebScanner` to just scan Chromium Documents